src/about-us/index.html

```diff
@@ -0,0 +1,189 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% block title -%}
+    About Us
+{%- endblock %}
+
+
+{% block content_main %}
+    <section class="block
+                    block__flush-top">
+        <h1>About Us</h1>
+        <p class="h3">
+            We're a new government agency created
+            after the financial crisis of 2008.
+            We write and enforce rules to ensure that markets for
+            consumer financial products and services are fair and open,
+            and to make financial information easier to understand.
+            We also build tools and resources that help arm consumers
+            with the information they need,
+            and empower consumers to make the best financial decisions.
+        </p>
+        <ul class="list list__links">
+            <li class="list_item">
+                <a href="/the-bureau/" class="list_link">
+                    Learn more about the Bureau
+                </a>
+            </li>
+            <li class="list_item">
+                {# TODO: Add link to Consumer Guides #}
+                <a class="list_link u-link__disabled">
+                    Find resources and tools for consumers
+                </a>
+            </li>
+            <li class="list_item">
+                <a href="/offices/payments-to-harmed-consumers/"
+                   class="list_link">
+                    Find out about our work to protect consumers
+                </a>
+            </li>
+        </ul>
+    </section>
+    <div class="content-l">
+        {# The extra div is to prevent border tapering from combining block__border and content-l classes #}
+        <div class="content-l_col-1-2">
+            <section class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-bottom">
+                <h2>CFPB Leadership</h2>
+                <div class="media bureau-bio u-mb30">
+                    <div class="media_image-container">
+                        <img class="media_image"
```

Comment:
- Create new class `media_image__125_to_80` in https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/static/css/media.less#L199, remove https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/static/css/pages/bureau.less#L73-L78

 _Ties into https://github.com/cfpb/cfgov-refresh/issues/722_

- Is 125px max size the right size? Can it be 130px or 150px for consistently with image sizes elsewhere?

- Top leadership images on this page are larger than needed. 360px image expects a max size of 180px.